cql3/lists.cc

```diff
@@ -182,7 +182,11 @@ lists::marker::bind(const query_options& options) {
 }
 
 expr::expression lists::marker::to_expression() {
-    throw std::runtime_error(fmt::format("to_expression not implemented! {}:{}", __FILE__, __LINE__));
+    return expr::bind_variable {
+        .shape = expr::bind_variable::shape_type::scalar,
```

Comment:
Is it really a scalar?

I expect not, but also that it doesn't matter, and the shape was used in deciding which marker subtype to create during prepare.

Reply:
Tbh I'm not sure about it, but a `scalar` was the closest thing to a list value.